packages/tenant-process/src/services/tenantService.ts

```diff
@@ -448,6 +448,54 @@ export function tenantServiceBuilder(
       await repository.createEvent(event);
       return updatedTenant;
     },
+    async revokeDeclaredAttribute({
+      attributeId,
+      organizationId,
+      correlationId,
+    }: {
+      attributeId: AttributeId;
+      organizationId: TenantId;
+      correlationId: string;
+    }): Promise<Tenant> {
+      logger.info(
+        `Revoking declared attribute ${attributeId} to ${organizationId}`
+      );
+      const requesterTenant = await retrieveTenant(
+        organizationId,
+        readModelService
+      );
+
+      const maybeDeclaredTenantAttribute = requesterTenant.data.attributes.find(
+        (attr) => attr.id === attributeId
+      );
+
+      if (
+        !maybeDeclaredTenantAttribute ||
+        maybeDeclaredTenantAttribute.type !== "PersistentDeclaredAttribute"
+      ) {
+        throw attributeNotFound(attributeId);
```

Comment:
Same as you did [here](https://github.com/pagopa/interop-be-monorepo/pull/381/files#diff-c52f69aae83f1d150aa67f380d299976f1a67b9cb76738048267062cdb848f3eR382-R385)

```suggestion
      const maybeDeclaredTenantAttribute = requesterTenant.data.attributes.find(
        (attr) => attr.id === attributeId && maybeDeclaredTenantAttribute.type !== "PersistentDeclaredAttribute"
      );

      if (!maybeDeclaredTenantAttribute) {
        throw attributeNotFound(attributeId);
```